app/src/main/java/com/quran/labs/androidquran/ui/fragment/TabletFragment.java

```diff
@@ -319,16 +320,18 @@ protected void onPostExecute(RectF[] rect) {
           if (mRightImageView != null && mLeftImageView != null) {
             mRightImageView.setPageBounds(rect[0]);
             mLeftImageView.setPageBounds(rect[1]);
-            Context context = getContext();
-            if (mOverlayText && context != null) {
```

Comment:
i intentionally added that null check because we were having crashes due to this, so please re-add this back.

because this is in `onPostExecute`, we can easily have a case where the fragment is killed before the `AsyncTask` finishes - when the `AsyncTask` is done, there is no `context` because the `Fragment` is gone, so we should just bail - by not doing this check, we'll crash in the next few lines where we try to use the context to get sura names, etc.